cartography/data/jobs/cleanup/aws_import_ec2_key_pairs_cleanup.json

```diff
@@ -0,0 +1,8 @@
+{
+  "statements": [{
+    "query": "MATCH (:AWSAccount{id: {AWS_ID}})-[:RESOURCE]->(n:EC2KeyPair) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+    "iterative": true,
+    "iterationsize": 100
+  }],
```

Comment:
Need a cleanup job for the `[:LOGIN_KEY]` relationship.  

Something like
```
"MATCH (:EC2KeyPair)-[r:LOGIN_KEY]->(:EC2Instance) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted"
```

Reply:
Added, except the relationship is now `[:SSH_LOGIN_TO]`

Reply:
Added. But `DETACH DELETE` will [delete a node and all it's relationships](https://neo4j.com/docs/cypher-manual/current/clauses/delete/#delete-delete-all-nodes-and-relationships). Why do we specifically also code in relationships to be deleted?

Reply:
There are cases where the node may continue to exist but the relationship between it and other nodes changes :-)

Removing stale relationships accounts for this case.

Reply:
Got it.